src/style.css

```diff
@@ -0,0 +1,55 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-position: center;
+  background-repeat: no-repeat;
+  background-image: url(images/star.svg);
+  margin-left: 4px;
+  width: 16px;
+  height: 16px;
+}
+
+.stars :nth-child(-n + 1) {
+  margin: 0;
+}
+
+.stars--1 :nth-child(-n + 1) {
+  background-image: url(images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
```

Comment:
* Avoid repeating styles, you can set background-position and background-repeat in one place
<img width="1109" alt="Screenshot 2023-08-23 at 16 29 23" src="https://github.com/mate-academy/layout_stars/assets/135232001/3eb13166-98b4-42be-9388-922a354eb11a">

* Consider following the checklist for the task